### ISSUE TITLE
intraventricular conduction delay

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7640]
**Preferred term label:**
intraventricular conduction delay -- how is this related to QRS widening?

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
611174 
PMID: 34899143  0/2
PMID: 22581230  4/4

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is a synonym, see https://litfl.com/intraventricular-conduction-delay-qrs-widening/
I will add it.

**Comment by pnrobinson** [comment]
added annotations.
